src/ui/templates/results/directanswer.hbs

```diff
@@ -1,3 +1,6 @@
+{{#if _config.defaultCard}}
```

Comment:
nit: do we want to be consistent with the hbs tabbing and tab everything inside the {{if}}{{else}} blocks?

Reply:
I've been starting to prefer not indenting after a handlebars helper but you're right should be consistent here